apps/ledger-live-desktop/src/newArch/Collectibles/components/index.tsx

```diff
@@ -0,0 +1,6 @@
+export { default as Skeleton } from "./Skeleton";
```

Comment:
Why not just use named exports in the files themsevles ?
And do `export { Skeleton } from "./Skeleton"`